packages/schema/src/schema/project/layers/stacLayer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "object",
+  "description": "StacLayer",
+  "title": "IStacLayer",
+  "required": ["data"],
+  "additionalProperties": false,
+  "properties": {
+    "data": {
+      "type": "object",
+      "description": "The data of the source"
+    },
+    "opacity": {
```

Comment:
Missing a visible property like other layers, so you cannot hide those layers from the layers panel